universe/System.cpp

```diff
@@ -270,6 +270,15 @@ StarType System::NextYoungerStarType() const {
     return StarType(m_star - 1);
 }
 
+int System::NumPlanets() const {
+    int retval = 0;
+    for (std::set<int>::const_iterator it = m_planets.begin(); it != m_planets.end(); ++it) {
+        if (*it != INVALID_OBJECT_ID)
```

Comment:
Is this extra check needed ever? (When?)
Alternatively, just return m_planets.size();